base_tier_validation/models/tier_validation.py

```diff
@@ -297,6 +315,35 @@ def _notify_review_requested(self, tier_reviews):
                     body=rec._notify_requested_review_body()
                 )
 
+    def _notify_review_requested_by_sequence(self, tier_review):
+        if hasattr(self, 'message_post') and \
+                hasattr(self, 'message_subscribe'):
+            for rec in self:
+                # Subscribe reviewers and notify
+                getattr(rec, 'message_subscribe')(
+                    partner_ids=tier_review.reviewer_ids.mapped(
+                        "partner_id").ids)
+                getattr(rec, 'message_post')(
+                    message_type='email',
+                    body=rec._notify_requested_review_body(),
+                    partner_ids=tier_review.reviewer_ids.mapped(
+                        "partner_id").ids)
+
+    def _notify_review_accepted_requested_by_sequence(self, tier_review):
+        if hasattr(self, 'message_post') and \
+                hasattr(self, 'message_subscribe'):
+            for rec in self:
+                getattr(rec, 'message_subscribe')(
```

Comment:
If you have checked the `message_subscribe`, you can do rec.message_subscribe, no need of getattr

Reply:
Sorry for late answer,
Yes, I wanted to do as you say. But most of the way in the code was using like that https://github.com/OCA/server-ux/blob/12.0/base_tier_validation/models/tier_validation.py#L263-L269 .